e2e/slashing_test.go

```
@@ -0,0 +1,180 @@
+		"--chain-id", chain.Config().ChainID,
+		"--home", chain.HomeDir(),
+		"--node", chain.GetRPCAddress(),
+		"--keyring-backend", keyring.BackendTest,
+		"-y",
+	}
+	_, _, err = chain.Exec(ctx, delegate, nil)
+	require.NoError(t, err)
+
+	// TODO: get to work so validator from this node gets slashed offline
+	// stop validator from participating in voting
+	chain.Validators[0].PauseContainer(ctx)
```

golangci-lint (1.62.2)

Error return value of `(*github.com/strangelove-ventures/interchaintest/v7/chain/cosmos.ChainNode).PauseContainer` is not checked

(errcheck)